Lib/ctypes/__init__.py

```diff
@@ -263,6 +263,8 @@ class c_void_p(_SimpleCData):
 
 class c_bool(_SimpleCData):
     _type_ = "?"
+c_bool.__ctype_le__ = c_bool.__ctype_be__ = c_bool
+_check_size(c_bool)
```

Comment:
Not disagreeing that this should be here, I'd prefer to keep unrelated changes out of a PR. Could you open a seperate PR that adds this? (No need for an issue or blurb entry on it, just the one line change should be fine; tag me and I'll add `skip issue` and `skip news`.)
```suggestion
```

Reply:
I appreciate the input but it's not really worth my time to split this single line out into a separate PR. Any real reason it can't ride along with this one?

Reply:
We don't really like to accept changes that are unrelated to the actual fix. You can leave it in, but expect pushback from other reviewers.